_posts/2021-09-29-release-1.14.0.md

```diff
@@ -163,12 +163,11 @@ Apache Flink buffers a certain amount of data in its network stack to be able to
 bandwidth of fast networks. A Flink application running with high throughput uses some (or
 all) of that memory. Aligned checkpoints flow with the data through the network buffers in milliseconds.
 
-When a Flink application becomes (temporarily) backpressured (for example when being backpressured
-by an external system, or when hitting skewed records), this typically results in a lot more data in
-the network buffers than is necessary utilize enough network bandwith for the application's current throughput
-(which is lowered due to backpressure). There is even an adverse effect: more buffered data means
-the checkpoints need to do more work. Aligned checkpoint barriers need to wait for more data to be
-processed, unaligned checkpoints need to persist more in-flight data.
+During (temporal) backpressure from a resource bottleneck such as an external system, data skew, or (temporarily)
+increased load, Flink was buffering a lot more data inside its network buffers than necessary to utilize
+enough network bandwith for the application's current – backpressured – throughput. This actually has an adverse
+effect: more buffered data means the checkpoints need to do more work: Aligned checkpoint barriers need to wait
+for more data to be processed, unaligned checkpoints need to persist more in-flight data.
```

Comment:
```suggestion
During (temporary) backpressure from a resource bottleneck such as an external system, data skew, or (temporarily)
increased load, Flink was buffering a lot more data inside its network buffers than necessary to utilize
enough network bandwidth for the application's current – backpressured – throughput. This actually has an adverse
effect because more buffered data means that the checkpoints need to do more work. Aligned checkpoint barriers need to wait for more data to be processed, unaligned checkpoints need to persist more in-flight data.
```